src/build/sign.rs

```diff
@@ -11,36 +11,39 @@ pub fn copy_sign(
     key: &armake2::sign::BIPrivateKey,
 ) -> Result<bool, Error> {
     let path = entry.path();
-    let cpath = path.clone();
-    let cpath = cpath.to_str().unwrap().replace(r#"\"#, "/");
-    let pbo_path = cpath.replace((folder.clone() + "/").as_str(), "");
-    if !path.ends_with(".pbo") && !pbo_path.contains(p.prefix.as_str()) {
+    let path_posix = path.clone().to_str().unwrap().replace(r#"\"#, "/");
+    let pbo_filename = path_posix.replace((folder.clone() + "/").as_str(), "");
+
+    if !path.ends_with(".pbo") && !pbo_filename.starts_with(p.prefix.as_str()) {
```

Comment:
This is where I integrated #51. I meant to separate it into a separate commit but apparently I forgot.